logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/connection/ObserveConnectionListUseCase.kt

```diff
@@ -0,0 +1,20 @@
+package com.wire.kalium.logic.feature.connection
+
+import com.wire.kalium.logic.data.connection.ConnectionRepository
+import com.wire.kalium.logic.data.user.Connection
+import com.wire.kalium.logic.sync.SyncManager
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.Flow
+
+@OptIn(ExperimentalCoroutinesApi::class)
+class ObserveConnectionListUseCase(
```

Comment:
We can extract an interface out of this `UseCase`, so we can mock them easily on ARloaded.
You could see an example of this in `AcceptConnectionRequestUseCase`.

It would be nice to have the kdocs also, so we can get it documented automatically by dokka 🙌

Reply:
Sure, I will remember that 🙏